src/components/VerboseTextFieldArray.tsx

```diff
@@ -89,6 +91,11 @@ export default function VerboseTextFieldArray(
             <FormHelperText>{descriptionParagraph}</FormHelperText>
           ))}
         </Grid>
+        {necessity && (
+          <FormHelperText sx={{ fontWeight: 800 }}>
+            * {necessity === "recommended" ? "Recommended" : "Required"}
+          </FormHelperText>
```

Comment:
It looks like this logic is duplicated. It could be lifted to a shared component that gets imported and reused. That component module could also export the shared type.

Reply:
addressed with 00392213269e850229f70062d38ff2091eacb30a